### PR TITLE
Add LandingpageView

### PR DESCRIPTION
**Pull request by Frlundsten**
Add a landingpage
JIRA PG2-19

### CODE REVIEW

File: frontend/src/components/__tests__/LandingView.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        expect(firstMovieTitle.text()).toBe('The Matrix')
+        expect(firstMovieDescription.text()).toContain('A computer hacker learns from mysterious rebels')
+        expect(firstMovieImgUrl).toBe('https://image.tmdb.org/t/p/w500/f89U3ADr1oiB1s9GkdPOEpXUk5H.jpg')
+
+    })
+
+    test('ArrayOfHighestRated does not contain Interstellar',()=>{
+        expect(wrapper.vm.arrayOfHighestRated.length).toBe(2)
+        expect(wrapper.vm.arrayOfHighestRated.every(movie => movie.name !== 'Interstellar')).toBeTruthy();
+    })
```
Comment (Mattan41) [inline frontend/src/components/__tests__/LandingView.spec.js:119]: This test fails for me  - AssertionError: expected 3 to be 2, and AssertionError: expected false to be truthy

**Review by Mattan41 — COMMENTED** [review]
looks good and works fine but test 'ArrayOfHighestRated does not contain Interstellar' fails for me

**Review by DevelopedByDanny — APPROVED** [review]
Nice looking page! :)

**Review by Mattan41 — APPROVED** [review]
looks good works fine, with dummydata added locally

**Pull request merged**